src/Uno.UWP/Helpers/DrawableHelper.Android.cs

```diff
@@ -33,8 +33,8 @@ public static Type Drawables
 		/// <param name="imageName">Name of the image</param>
 		/// <returns>Resource's id</returns>
 		public static int? FindResourceId(string imageName)
```

Comment:
I've been looking at compatibility, and this method is unfortunately used in ways that require the name to be just the file name.

We should do two things, adjust this method so that it searches through all resources using an "EndsWith" on keys or maybe lazily create a second dictionary that only uses the last part of a `Split('_')` for performance) using just the file name without the path, and make a second method, internal this time, which takes a full path.

That method was public a long time ago (you probably remember it ;)) and obviously should not have been...

Reply:
How do we want to deal with collisions when we have two images with the same name but they were just in different folders in the case of the new dictionary? Just take the first one?

Reply:
There's also the issue of when an image already has a underscore in its name, this new drawables lookup dictionary will have the wrong key

Reply:
Unless we want to encode the folder hierarchy in some wacky way that normal files wouldn't be named? But I know there's a lot of restrictions around resource names for Android

Reply:
That's fair... For duplicates and duplicate paths, it was not working before in all cases and we can detail these as a restrictions in the documentation.

In all cases, this method should not be used and be marked as non-browsable.